src/api/goods.ts

```diff
@@ -10,10 +10,13 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => goods.sort(
+      (a, b) => a.name.localeCompare(b.name),
+      // eslint-disable-next-line
+    ).filter(({}, index) => index < 5));
```

Comment:
```suggestion
    ).filter((_, index) => index < 5));
```
or give another name